snap/local/build-helpers/bin/set-version.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash -e
+
+if [ -z "$SNAPCRAFT_PROJECT_DIR" ]; then
+    echo "Error: SNAPCRAFT_PROJECT_DIR not set!"
+    exit 1
+fi
+
+pushd $SNAPCRAFT_PROJECT_DIR
+
+if git describe ; then
+    # get the short tag without commit count and hash when there are untagged commits
+    TAG=$(git describe --tags --abbrev=0) 
+    # drop the v prefix
+    VERSION=$(echo $TAG | sed 's/v//')
```

Comment:
This can also be done by: $(echo "${TAG/v/}")

Parameter expansion can do replacement with less overhead.

Reply:
Oh, I just tested this substitution doesn't work in snapcraft.